docs/DatabaseCatalogAPI.yaml

```diff
@@ -473,28 +506,27 @@ components:
             note:
               description: A note to clarify complex use cases for consumers.
               type: string
-            download_date:
+            downloaded_at:
+              description: The date and time the dataset was downloaded from the producer, in ISO 8601 format.
               type: string
-              example: 2020-12-31
+              example: 2023-07-10T22:06:00Z
               format: date
-            creation_date:
+            created_at:
```

Comment:
I thought `created_at` and `updated_at` were going to be removed, since a new dataset is created any time it's updated.

Reply:
done.